scripts/west_commands/runners/core.py

```diff
@@ -225,24 +225,31 @@ class RunnerCaps:
       needed by SoCs which have flash-like areas that can't be sector
       erased by the underlying tool before flashing; UICR on nRF SoCs
       is one example.)
+
+    - erase: whether the runner supports a --tool-opt (-O) option, which
```

Comment:
Did you mean to write `tool_opt:` here?

Reply:
> Did you mean to write tool_opt: here?

Yes, thank you. Fixed that and removed a bit of debugging leftovers in the latest push.